src/scripts/main.js

```diff
@@ -354,7 +354,40 @@
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+'use strict';
+
+const table = document.querySelector('.dashboard');
+
+function createCell(content) {
+  const cell = document.createElement('td');
+  cell.textContent = content;
+  return cell;
+}
+
+function calculateAge(person) {
+  return person.died - person.born;
+}
+
+function calculateCentury(year) {
+  return Math.ceil(year / 100 - 1);
+}
+
+function createRow(person) {
+  const row = document.createElement('tr');
+
+  const nameCell = createCell(person.name);
+  const sexCell = createCell(person.sex);
+  const bornCell = createCell(person.born);
+  const diedCell = createCell(person.died);
+  const ageCell = createCell(calculateAge(person));
+  const centuryCell = createCell(calculateCentury(person.died));
+
+  row.append(nameCell, sexCell, bornCell, diedCell, ageCell, centuryCell);
+
+  return row;
+}
 
-// write your code here
+people.forEach(person => {
+  const row = createRow(person);
+  table.appendChild(row);
+});
```

Comment:
The code that creates the table rows is clear and concise. However, it could be made more efficient. Currently, the table is reflowed each time a row is appended. This can be a costly operation in terms of performance, especially if the `people` array is large. Consider creating a document fragment, appending the rows to the fragment, and then appending the fragment to the table. This way, the table is reflowed only once, regardless of the number of rows.